rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4032,11 +4032,9 @@ bool opcode_016b(const ScriptArguments& args) {
     @arg coord Coordinates
     @arg angle Angle
 */
-void opcode_016c(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat angle) {
-    RW_UNIMPLEMENTED_OPCODE(0x016c);
-    RW_UNUSED(coord);
-    RW_UNUSED(angle);
-    RW_UNUSED(args);
+void opcode_016c(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat heading) {
+	coord = script::getGround(args, coord);
+    args.getState()->addHospitalRestart(glm::vec4(coord, heading));
```

Comment:
Mixing tabs and spaces?

Reply:
yes, I can fix these exact lines, but whole file should be reformatted anyway